pkg/volume/azure_file/azure_file.go

```diff
@@ -139,6 +142,42 @@ func (plugin *azureFilePlugin) newUnmounterInternal(volName string, podUID types
 	}}, nil
 }
 
+func (plugin *azureFilePlugin) RequiresFSResize() bool {
+	return false
+}
+
+func (plugin *azureFilePlugin) ExpandVolumeDevice(
+	spec *volume.Spec,
+	newSize resource.Quantity,
+	oldSize resource.Quantity) (resource.Quantity, error) {
+
+	if spec.PersistentVolume != nil && spec.PersistentVolume.Spec.AzureFile == nil {
```

Comment:
better be `spec.PersistentVolume==nil || ...`

Otherwise if spec.PersistentVolume is nil, Line#157 will raise null reference exception.